PYME/recipes/measurement.py

```diff
@@ -471,7 +471,7 @@ def execute(self, namespace):
         from PYME.Analysis.points import DistHist
         
         pos0 = namespace[self.inputPositions]
-        pos1 = namespace[self.inputPositions2 if self.inputPositions2 is not '' else self.inputPositions]
+        pos1 = namespace[self.inputPositions2 if self.inputPositions2 != '' else self.inputPositions]
```

Comment:
not sure where this change came from  ..., should be functionally equivalent though